### ISSUE TITLE
Add accessibility topic

### ISSUE THREAD

**Issue by gregvanl** [Microsoft/vscode-docs#144]
This should cover the accessibility features of VS Code:
- Zoom (session and persisted)
- High Contrast Themes
- Keyboard navigation and Tab trapping (Ctrl+M)
- Screen reader support

**Comment by chrisdias** [comment]
Current known issues:

Windows Limitations
* Cannot use keyboard (right, left arrow keys) to move between top level menu items, for example between File and Edit and View ... https://github.com/atom/electron/issues/2504

Linux Limitations
* 

OSX Limitations
* No screen reader support for editor

**Comment by gregvanl** [comment]
Live on code.visualstudio.com
